packages/EtendoHookBinder/src/screens/Metadata.tsx

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useState } from 'react';
+import styles from './styles.module.css';
+import { useWindow } from '../hooks/useWindow';
+
+export default function MetadataTest() {
+  const [windowId, setWindowId] = useState('100');
```

Comment:
The window with ID 100 is the tables and columns window, the idea is to start with the "Sales Order" window which is window 143.